### DISCUSSION TITLE
"can't evaluate field Height in type string" after updating doks-core from 1.0.7 to 1.1.2

### DISCUSSION THREAD

**Discussion by Yvand** [thuliteio/doks#1136] [Q&A] [answered]
After upgrading @hyas/doks-core from 1.0.7 to 1.1.2, I get this error when I run `npm run dev` :

```
Error: error building site: "/home/yvand/dev/hugo/Hugo-EntraCP/content/docs-azurecp/help/fix-setup-issues.md:91:1": failed to render shortcode "img": failed to process shortcode: "/home/yvand/dev/hugo/Hugo-EntraCP/node_modules/@hyas/images/layouts/shortcodes/img.html:125:4": execute of template failed: template: shortcodes/img.html:125:4: executing "shortcodes/img.html" at <partial "img.html" $ctx>: error calling partial: "/home/yvand/dev/hugo/Hugo-EntraCP/node_modules/@hyas/images/layouts/partials/img.html:203:13": execute of template failed: template: partials/img.html:203:13: executing "partials/img.html" at <$bi.Height>: can't evaluate field Height in type string
```

I am fairly certain this is a dumb configuration error on images shortcodes, but I cannot find how to correct it.
Link to the branch where error happens: https://github.com/Yvand/Hugo-EntraCP/tree/bump-doks

Thank you

**Comment by h-enk** [comment, marked as answer]
Please check your [configurations settings](https://getdoks.org/docs/start-here/upgrade-to-v1/#configure-the-integration)

**Reply by Yvand** [reply]
many things were wrong, but at the end it was 'simply' because the link to images was `assets/images/people-picker-EntraCP.png` instead of `images/people-picker-EntraCP.png`
